sound/soc/sof/sof-audio.h

```diff
@@ -180,6 +181,8 @@ struct sof_ipc_tplg_widget_ops {
  * @set_up_all_pipelines: Function pointer for setting up all topology pipelines
  * @tear_down_all_pipelines: Function pointer for tearing down all topology pipelines
  * @parse_manifest: Optional function pointer for ipc4 specific parsing of topology manifest
+ * @dai_load: Function pointer for ipc4 specific initialization of spcm data
+ * @dai_unload: Function pointer for ipc4 specific free of spcm data
```

Comment:
is this optional or not? the code shows it is, but there's no comment to that effect here.